components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Edge.Publisher/src/Engine/WriterGroupMessageSource.cs

```diff
@@ -216,7 +222,7 @@ private sealed class DataSetWriterSubscription : IDisposable {
                     var sequenceNumber = (uint)Interlocked.Increment(ref _currentSequenceNumber);
                     var snapshot = await Subscription.GetSnapshotAsync().ConfigureAwait(false);
                     if (snapshot != null) {
-                        CallMessageReceiverDelegates(this, sequenceNumber, snapshot);
+                        CallMessageReceiverDelegates(this, sequenceNumber, snapshot, false /* ??? */);
```

Comment:
>/* ??? */ [](start = 91, length = 9)

you want to keep the comment?

Reply:
I'm removing this parameter in a subsequent PR, but yes I will remove the comment.